### ISSUE TITLE
Cannot run with Tensorflow 2

### ISSUE THREAD

**Issue by nikhilparmar** [michhar/maskrcnn-custom#4]
Hello,
what changes do I need to make in order to run the files with Tensorflow 2.0 as its not running in TF2 by showing the error for Placeholder.

Please guide.
Thank YOu

**Comment by michhar** [comment]
Hi,

Thanks for your question!  I would very much like to upgrade to TF 2, but it may take some time.  If you would like to, I found the TF 1 to TF 2 conversion script at https://www.tensorflow.org/guide/upgrade.  I also found a nice article explaining the change with regard to Keras at https://www.pyimagesearch.com/2019/10/21/keras-vs-tf-keras-whats-the-difference-in-tensorflow-2-0/.

Given time I may try this as well and will add comments here.  If you discover how to do so, feel free to discuss here or create a PR.

P.S. I just made some small changes to the code this morning to add epochs as an arg and some image augmentation in the `custom.py` file.

Best Regards,
Micheleen
